android/src/main/java/com/mapbox/mapboxgl/MapboxMapOptionsSink.kt

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
I converted the file simple file to Kotlin for a better overview, which types could be optional.